C/CryptoMiniSat/build_tarballs.jl

```diff
@@ -6,20 +6,19 @@ const YGGDRASIL_DIR = "../.."
 include(joinpath(YGGDRASIL_DIR, "platforms", "mpi.jl"))
 
 name = "CryptoMiniSat"
-version = v"5.8.2"
-cryptominisat_version = v"5.8.0"
+version = v"5.11.4"
+cryptominisat_version = v"5.11.4"
```

Comment:
It doesn't look like this is used at all, or am I missing something?

Reply:
Indeed, I don't think it is. I just left it from the previous file.